doc/from-zero-to-hacking.md

```diff
@@ -234,6 +234,18 @@ Aquarium. The next section will cover development in finer detail.
 
 Let's add our first trivial patch!
 
+### Access the Ceph cluster via CLI
+
+For developing purposes it's sometimes helpful to access the cluster via
+CLI. You can reach the Ceph CLI within Aquarium as follows:
+
+1. Connect to the VM and run Aquarium as explained above
+
+2. `sudo su` within the VM
+
+3. `cephadm shell`
```

Comment:
this could be simply a `sudo cephadm shell`.

Reply:
Updated the PR accordingly.